packages/modular-scripts/src/esbuild-scripts/start/plugins/wsReload.ts

```diff
@@ -13,43 +14,44 @@ export default function createPlugin(
     setup(build) {
       let building = false;
       let result: esbuild.BuildResult;
-      const { appPath } = paths;
 
-      const publish = () => {
-        // eslint-disable-next-line @typescript-eslint/no-misused-promises
-        server.clients.forEach(async (socket) => {
-          socket.send(
-            JSON.stringify({
-              name,
-              building,
-              result: {
-                errors: await Promise.all(
-                  result.errors.map((error) => formatError(error, appPath)),
+      const publishClient = async (socket: Websocket) => {
+        socket.send(
+          JSON.stringify({
+            name,
+            building,
+            result: {
+              errors: await Promise.all(
+                result.errors.map((error) =>
+                  formatError(error, paths.modularRoot),
                 ),
-                warnings: await Promise.all(
-                  result.warnings.map((warning) =>
-                    formatError(warning, appPath),
-                  ),
+              ),
+              warnings: await Promise.all(
+                result.warnings.map((warning) =>
+                  formatError(warning, paths.modularRoot),
                 ),
-              },
-            }),
-          );
-        });
+              ),
+            },
+          }),
+        );
       };
 
-      server.on('connection', () => {
-        publish();
-      });
+      const publishAll = () => {
+        // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Is there a way to resolve these lint warnings neatly?

Reply:
Maybe

```
return Promise.all(server.clients.map(publishClient));
```

Reply:
That and a bunch of `void` typings did the trick.